app/assets/stylesheets/pages/banner-intro.scss

```diff
@@ -7,13 +7,16 @@
 }
 
 .banner-intro-intro {
+  padding-top: 9px;
   font-size: 17px;
 }
 
 .banner-intro-tagline {
   padding-right: 30px;
+  padding-top: 30px;
   padding-left: 30px;
-  margin-bottom: 60px;
+  margin-bottom: 50px;
+  font-size: 1.23rem;
```

Comment:
It seems weird we're mixing `rem` and `px`. I'd honestly prefer rem for the entire project but this can be a discussion/task that we can do later

Reply:
Agreed, these are under the umbrella of low-priority refactors I'd still like to do